contracts/settling_game/tokens/Resources_ERC1155_Mintable_Burnable.cairo

```diff
@@ -26,7 +26,7 @@ from contracts.token.library import ERC1155
 func initializer{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
     uri : felt, proxy_admin : felt
```

Comment:
In the PR description, you mention  "Initializer no longer takes a URI input" but here it still does.